src/crystal/dwarf/line_numbers.cr

```diff
@@ -240,6 +282,100 @@ module Crystal
         end
       end
 
+      record LNCTFormat,
+        lnct : LNCT,
+        format : FORM
+
+      enum LNCT : UInt32
```

Comment:
Maybe add some documentation, similar to the other structs and methods above? To explain what LNCT is, that it's new in DWARF 5, and a reference to the section that defines the format in the DWARF 5 std, ...